src/react-app-env.d.ts

```diff
@@ -1,2 +1,28 @@
 // eslint-disable-next-line
 /// <reference types="react-scripts" />
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: {};
+  phone: string;
+  website: string;
+  company: {};
+}
+
+interface Todo {
+  userId: number;
+  id: number;
+  title: string;
+  completed: boolean;
+}
+
+interface GetTodos {
```

Comment:
GetTodos? how do you get something with it?

Reply:
You are right